Makefile

```diff
@@ -71,14 +73,14 @@ $(ISORT_VIRTUALENV): $(PY34)
 	$(PY34) -m virtualenv $(ISORT_VIRTUALENV)
 
 format: $(PYFORMAT) $(ISORT)
-	$(TOOL_PYTHON) scripts/enforce_header.py
+	$(FILES_TO_FORMAT) | $(TOOL_PYTHON) scripts/enforce_header.py
 	# isort will sort packages differently depending on whether they're installed
 	$(ISORT_VIRTUALENV)/bin/python -m pip install django pytz pytest fake-factory numpy flaky
-	find src tests examples -name '*.py' | xargs  env -i \
+	$(FILES_TO_FORMAT) | xargs  env -i \
```

Comment:
Is the double space after `xargs` deliberate? If not, might be a good time to tidy that up.